### PR TITLE
Launch empty world fix

### PR DESCRIPTION
**Pull request by fmessmer**
finalizing #120

### CODE REVIEW

**Comment by fmessmer** [comment]
@ipa-mig could the nav-team please test whether the environments spawn/behave as expected (joint_states) and the laser-bug does not occur?

File: cob_bringup_sim/CMakeLists.txt [hunk 1]
```diff
@@ -15,7 +15,10 @@ set(robotlist
   cob4-2
   cob4-3
   cob4-4
+  cob4-5
   cob4-6
+  cob4-7
+  cob4-10
```
Comment (mgruhler) [inline cob_bringup_sim/CMakeLists.txt:21]: just out of curiousity: what about cob4-8 and 9? :-D

File: cob_gazebo_worlds/urdf/empty/empty.urdf.xacro [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro"
+       name="empty">
+
+  <link name="world"/>
```
Comment (mgruhler) [inline cob_gazebo_worlds/urdf/empty/empty.urdf.xacro:5]: So empty does relly only spawn the world link and not even a floor? Or did I miss something?

**Review by mgruhler — APPROVED** [review]

**Comment by mgruhler** [comment]
Travis complains about: `	cannot find node [fake_monitoring.py] in package [cob_controller_configuration_gazebo]`

File: cob_gazebo_worlds/urdf/empty/empty.urdf.xacro (hunk 2 above)
Comment (fmessmer) [reply cob_gazebo_worlds/urdf/empty/empty.urdf.xacro:5]: the floor comes with the ```empty.world``` passed to the ```gazebo_ros``` launch file [here](https://github.com/ipa-fxm/cob_simulation/blob/89efae492b1461ecd071c9df9026fc27e250a40d/cob_gazebo_worlds/launch/world.launch#L27)

**Comment by fmessmer** [comment]
install tag was missing - added in https://github.com/ipa320/cob_robots/pull/562

File: cob_bringup_sim/CMakeLists.txt (hunk 1 above)
Comment (fmessmer) [reply cob_bringup_sim/CMakeLists.txt:21]: no idea about the naming policy...

**Comment by mgruhler** [comment]
@ipa-srd @ipa-jba does any one of you have a student who could do the testing?

**Comment by stefandoerr** [comment]
@srd-fe could you do that please?
What you need to do:
1. Check out this branch
2. start simulation with
  a.) `ROBOT_ENV=empty` and see if everything is spawned correctly/behaves like it should
  b.) `ROBOT_ENV=ipa-apartment`, again have a look if everything is fine, especially if the laser scans are fine, i.e. if you look at them with rviz they are correctly ending at the walls and not jumping from max range to wall and so on. Please do that test several times (maybe reboot once and test again)

**Comment by mgruhler** [comment]
@srd-fe also please shortly test the other envs, especially the `ipa_production_plant`

**Comment by fmessmer** [comment]
also please check whether parts are moving/can be moved as before, e.g. conveyor-belt
also whether joint_states and other topics, e.g. static laser scan, are received as before

(if this is important for the nav-team)

**Comment by fmessmer** [comment]
has this been tested by some nav-team-member/student?
@ipa-mig @ipa-srd @ipa-jba @ipa-flg @srd-fe

**Comment by fmessmer** [comment]
fixes #119

**Review by NadiaHG — APPROVED** [review]

**Comment by fmessmer** [comment]
I haven't heard any complains yet....thus, I'm merging...
@nav-team: report in a new issue in case behavior is not as expected

**Pull request merged**